backend/tests.py

```diff
@@ -122,6 +122,12 @@ def test_base_endpoint(self):
         for endpoint in endpoints:
             self.assertEqual(response[endpoint[0]], endpoint[1])
 
+    def get_field_from_response(self, response, name):
```

Comment:
Yay! Super helpful to be able to do this with the new questionnaire meta structure